### ISSUE TITLE
Cmake Qt5Quick not found although the .so file is available

### ISSUE THREAD

**Issue by code-hunger** [SFTtech/openage#851]
Here's the error message: https://pastebin.com/zhZAUwqe

It's weird because `find /usr -iname '*qt5quick*'` shows that I have the shared object files (e.g. `/usr/lib/libQt5Quick.so.5.8.0`). I have installed them via the systems package manager.

I searched also for `Qt5QuickConfig.cmake` and `qt5quick-config.cmake` (again, in /usr) without any results.
Google gives very few results and I couldn't solve it myself. Any ideas?

**Comment by Vtec234** [comment]
Usually, the shared library files are not enough, since you also need the development files - that is, a static library shim to link with and the sources. Which distribution are you on? Most distributions have a package named something like `qtquick-devel` or `qtquick-dev` that contains these.

**Comment by code-hunger** [comment]
Running Void linux, here only `qt5-quickcontrols` and `qt5-quickcontrols2` exists as packages which contain the keyword `quick` inside them (and I have installed these packages already)

**But!** I just did it... while writing this post. I searched for the package owning the `.so` files - it turned to be `qt5-declarative`. I installed the corresponding `-devel` package and it worked!
Happily compiling now.
